packages/terra-application-header/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra App Delegate
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-app-delegate.svg)](https://www.npmjs.org/package/terra-app-delegate)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The AppDelegate is an object that provides a set of functions used to facilitate communcation between Containers and
+their consuming Containers/Applications.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-app-delegate/docs)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-app-delegate`
```

Comment:
Need to update terra-app-delegate to terra-application-header in the install instructions.